src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/AutoplayVuap.java

```diff
@@ -190,21 +213,25 @@ private WebElement findCloseButton() {
     return wait.forElementClickable(By.cssSelector(String.format(CLSE_BUTTON_SELECTOR_FORMAT, slot)));
   }
 
-  private void waitFor(final Predicate<AutoplayVuap> predicate, final long timeout) {
-    new FluentWait<>(this)
-        .withTimeout(timeout, TimeUnit.SECONDS)
-        .pollingEvery(1, TimeUnit.SECONDS)
-        .until(predicate);
-  }
-
-  private boolean isOverlayVisible() {
-    return driver.findElement(By.cssSelector(String.format(REPLAY_BUTTON_SELECTOR_FORMAT, slot))).isDisplayed();
+  private boolean isOverlayNoVisible() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "isOverlayNoVisible" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)